translations/portuguese.json

```diff
@@ -39,9 +38,9 @@
 		"DATA COMMANDS": "COMANDOS DE DADOS",
 		"DRIVER COMMANDS": "COMANDOS DE DRIVER",
 		"MODERATOR COMMANDS": "COMANDOS DE MODERATOR",
-		"LEADER COMMANDS": "COMANDOS DE LEADER",
+		"ADMIN COMMANDS": "COMANDOS DE LEADER",
```

Comment:
COMANDOS DE ADMINISTRATOR
(direct copy from the english just like the other ranks)